caiman/source_extraction/cnmf/utilities.py

```diff
@@ -1159,14 +1159,19 @@ def get_file_size(file_name, var_name_hdf5='mov'):
                 else:
                     T, dims = siz[0], siz[1:]
             elif extension in ('.avi', '.mkv'):
-                cap = cv2.VideoCapture(file_name)
-                dims = [int(cap.get(cv2.CAP_PROP_FRAME_HEIGHT)), int(cap.get(cv2.CAP_PROP_FRAME_WIDTH))]
-                T = int(cap.get(cv2.CAP_PROP_FRAME_COUNT))
-                if dims[0] <= 0 or dims[1] <= 0 or T <= 0:
-                    # fallback to pims, like load() and load_iter() do
-                    pims_movie = pims.Video(file_name)
-                    T = len(pims_movie)
-                    dims[0], dims[1] = pims_movie.frame_shape[0:2]
+                if 'CAIMAN_LOAD_AVI_FORCE_FALLBACK' in os.environ:
+                        pims_movie = pims.PyAVReaderTimed(file_name) # duplicated code, but no cleaner way
+                        T = len(pims_movie)
+                        dims[0], dims[1] = pims_movie.frame_shape[0:2]
```

Comment:
`dims[n]` is called before it is defined will throw error. Fix with something like:

```
height, width = pims_movie.frame_shape[0:2]
dims = [height, width]
```